app/uk/gov/nationalarchives/omega/editorial/config/Config.scala

```diff
@@ -23,7 +23,9 @@ package uk.gov.nationalarchives.omega.editorial.config
 
 case class Config(
   broker: HostBrokerEndpoint,
-  credentials: UsernamePasswordCredentials
+  credentials: UsernamePasswordCredentials,
+  defaultRequestQueueName: String,
+  legalStatusRequestQueueName: Option[String] = None
```

Comment:
I understand this allows to switch between the data sources to use either the frontend or the services layer. Does this mean there would be different queue name(s) for the request endpoints in the API services?

Reply:
Yes, exactly. I created a diagram which I posted [here](https://national-archives.atlassian.net/wiki/spaces/PACT/pages/51773453/Service-based+Queue+Mapping). Hopefully that will make it a bit clearer.